pyiceberg/table/__init__.py

```diff
@@ -254,6 +254,13 @@ def property_as_bool(properties: Dict[str, str], property_name: str, default: bo
             return value.lower() == "true"
         return default
 
+    @staticmethod
+    def get_first_property_value(properties: Properties, *property_names: str) -> Optional[Any]:
+        for property_name in property_names:
+            if property_value := properties.get(property_name):
+                return property_value
+        return None
+
```

Comment:
This may be in a follow-up PR: I am considering whether it is worth moving the `PropertyUtil` class to a module in utils. Currently, PropertyUtil is used in the table, catalog, and io modules. Given its wide usage and the fact that it is a standalone utility class, I believe it can be extracted from the table module.

Reply:
Agreed. We can probably reorganize many of our functions to avoid circular dependencies 🙂